devlab/config.template

```diff
@@ -40,6 +40,7 @@ ext_cidr = <src_ext_cidr>
 user = <src_user>
 password = <src_password>
 tenant = <src_tenant>
+service_tenant = <src_name_of_tenant_for_the_openstack_services>
```

Comment:
Please put actual value which is `service` for devlab. I don't think we need it configurable right now.